.github/workflows/lintcommit.yml

```diff
@@ -18,4 +18,4 @@ jobs:
       - uses: rhysd/action-setup-vim@v1
         with:
           neovim: true
-      - run: nvim --clean -es +"lua require('scripts.lintcommit').main({trace=false})"
+      - run: nvim --clean -l scripts/lintcommit.lua main -trace
```

Comment:
If we change this to have `trace` enabled by default, we might as well remove the option and just be verbose always.

Reply:
It's not enabled `-trace` disables, `+trace` enables. We can use a different syntax if you prefer?

Reply:
`--trace` and  `--no-trace` is more standard in (shell) scripts?

Reply:
Sure

Reply:
But no strong opinions either way, I just didn't understand the syntax.